packager/transformer.js

```diff
@@ -37,6 +37,21 @@ function transform(src, filename, options) {
   }
   config.plugins = extraPlugins.concat(config.plugins);
 
+  // Manually resolve all default Babel plugins. babel.transform will attempt to resolve
+  // all base plugins relative to the file it's compiling. This makes sure that we're
+  // using the plugins installed in the react-native package.
+  config.plugins = config.plugins.map(function(plugin) {
+    // Normalise plugin to an array.
+    if (!Array.isArray(plugin)) {
+      plugin = [plugin];
+    }
```

Comment:
Each entry in the plugins array can be a plugin or a [plugin, options] pair. In either case, the plugin can be the name of a plugin or the plugin itself.